toplevel/coqargs.ml

```diff
@@ -526,6 +528,12 @@ let parse_args arglist : coq_cmdopts * string list =
     |"-color" -> set_color oval (next ())
     |"-config"|"--config" -> { oval with print_config = true }
     |"-debug" -> Coqinit.set_debug (); oval
+    |"-diffs" -> let opt = next () in
+                  if List.exists (fun x -> opt = x) ["off"; "on"; "removed"] then
+                    Proof_diffs.write_diffs_option opt
+                  else
+                    (prerr_endline ("Error: on|off|removed expected after -diffs"); exit 1);
+                  { oval with diffs_set = true }
```

Comment:
Why not put the string in the oval and use it later?

Reply:
I added a "Set Diffs on|off|removed" command.  Such option-setting commands require a setter routine such as write_diffs_option.  If opts is available as a modifiable global value then I could store the option setting there.

Reply:
@SkySkimmer Do you have any other review comments?  If this looks OK now, would you approve the changes?  Thanks.